docs/docs/code/bash/README.md

```diff
@@ -0,0 +1,199 @@
+# Writing Bash in Code Steps
+
+Prefer to write quick scripts in bash? We've got you covered.
+
+You can run any bash in a Pipedream step within your workflows.
+
+::: warning
+Bash steps are available in a limited alpha release.
+
+You can still run arbitrary bash scripts, including [sharing data between steps](/code/bash/#sharing-data-between-steps) as well as [accessing environment variables](/code/bash/#using-environment-variables).
+
+However, features available in [Node.js steps](/code/nodejs) like `$.respond`, `$.end`, and `$.auth` are not yet available in bash. If you have any questions please [contact support](https://pipedream.com/support).
```

Comment:
I would recommend reframing this section to talk about the features in the abstract, instead of the specific methods on the `$`. Maybe something like

> However, you can't connect accounts, return HTTP responses, or take advantage of other features available in the Node.js environment at this time.

Maybe also add "If you have any questions **, find any bugs, or have any feedback** ..." in the last sentence.

Reply:
@ctrlaltdylan did you see the rest of these comments in this file?